### PR TITLE
Query : PeerTotalSent

### PR DESCRIPTION
**Pull request by parv0888**
## Purpose
Adds support for query `PeerTotalSent`

## Changes
Adds support for query `PeerTotalSent`

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [x] (If necessary) I have updated the CHANGELOG.

## CLA acceptance
By submitting the contribution I accept the terms and conditions of the
Contributor License Agreement v1.0
- link: https://developers.concordium.com/CLAs/Contributor-License-Agreement-v1.0.pdf

- [x] I accept the above linked CLA.

### CODE REVIEW

File: concordium-sdk/src/main/java/com/concordium/sdk/Client.java [hunk 1]
```diff
@@ -245,6 +245,14 @@ public CryptographicParameters getCryptographicParameters(Hash blockHash) throws
         return cryptographicParameters;
     }
 
+    /**
+     * Gets the total No of sent packets.
```
Comment (EmilLa1) [inline concordium-sdk/src/main/java/com/concordium/sdk/Client.java:249]:
```suggestion
     * Gets the total number of packets sent.
```

File: concordium-sdk/src/main/java/com/concordium/sdk/Client.java [hunk 2]
```diff
@@ -245,6 +245,14 @@ public CryptographicParameters getCryptographicParameters(Hash blockHash) throws
...
+     * @return Total no of Sent Packets.
```
Comment (EmilLa1) [inline concordium-sdk/src/main/java/com/concordium/sdk/Client.java:250]:
```suggestion
     * @return Total number of packets sent.
```

**Review by EmilLa1 — CHANGES_REQUESTED** [review]
Looks good. :+1: 

Minor comments. 

Can you add it to the appropriate section of the README.md also?
Here
https://github.com/Concordium/concordium-java-sdk/blob/main/README.md#fill-in
and here
https://github.com/Concordium/concordium-java-sdk/blob/main/README.md#fill-in-1

File: concordium-sdk/src/main/java/com/concordium/sdk/Client.java (hunk 1 above)
Comment (parv0888) [reply concordium-sdk/src/main/java/com/concordium/sdk/Client.java]: done

File: concordium-sdk/src/main/java/com/concordium/sdk/Client.java (hunk 2 above)
Comment (parv0888) [reply concordium-sdk/src/main/java/com/concordium/sdk/Client.java]: done

**Review by EmilLa1 — APPROVED** [review]
Looks good. 👍 
Please rebase so we can get it merged.

**Pull request merged**